server/db/driver/pg/accounts.go

```diff
@@ -67,6 +74,22 @@ func (a *Archiver) Accounts() ([]*db.Account, error) {
 	return accts, nil
 }
 
+// AccountInfo returns data for an account.
+func (a *Archiver) AccountInfo(aid account.AccountID) (*db.Account, error) {
+	stmt := fmt.Sprintf(internal.SelectAccountInfo, a.tables.accounts)
+	acct := new(db.Account)
+	var accountID, pubkey, feeCoin []byte
+	var brokenRule byte
+	if err := a.db.QueryRow(stmt, aid).Scan(&accountID, &pubkey, &acct.FeeAddress, &feeCoin, &brokenRule); err != nil {
+		return nil, err
+	}
+	copy(acct.AccountID[:], accountID)
+	acct.Pubkey = dex.Bytes(pubkey)
+	acct.FeeCoin = dex.Bytes(feeCoin)
```

Comment:
The explicit `dex.Bytes` conversion is superfluous given the definition `type dex.Bytes []byte`.  That is, `a.Pubkey = pubkey` is good.